lib/libutee/include/tee_api_defines.h

```diff
@@ -331,17 +331,23 @@
 #define __OPTEE_ALG_ECDH_P384			0x80004042
 #define __OPTEE_ALG_ECDH_P521			0x80005042
 
-/* TEE_ALG_ECDSA_P* and TEE_ALG_ECDH_P* are deprecated */
-#define TEE_ALG_ECDSA_P192			TEE_ALG_ECDSA_SHA1
-#define TEE_ALG_ECDSA_P224			TEE_ALG_ECDSA_SHA224
-#define TEE_ALG_ECDSA_P256			TEE_ALG_ECDSA_SHA256
-#define TEE_ALG_ECDSA_P384			TEE_ALG_ECDSA_SHA384
-#define TEE_ALG_ECDSA_P521			TEE_ALG_ECDSA_SHA512
-#define TEE_ALG_ECDH_P192		TEE_ALG_ECDH_DERIVE_SHARED_SECRET
-#define TEE_ALG_ECDH_P224		TEE_ALG_ECDH_DERIVE_SHARED_SECRET
-#define TEE_ALG_ECDH_P256		TEE_ALG_ECDH_DERIVE_SHARED_SECRET
-#define TEE_ALG_ECDH_P384		TEE_ALG_ECDH_DERIVE_SHARED_SECRET
-#define TEE_ALG_ECDH_P521		TEE_ALG_ECDH_DERIVE_SHARED_SECRET
+/*
+ * TEE_ALG_ECDSA_P* and TEE_ALG_ECDH_P* are deprecated on API v1.2 or later
```

Comment:
Why is it important to get rid of these?
 What is the harm of keeping TEE_ALG_ECDSA_P* and TEE_ALG_ECDH_P* defines regardless of __OPTEE_CORE_API_COMPAT_1_1?

See also the comment above at lines 26-30.

Reply:
I propose this to prevent using using the deprecated macros since they change of value (now here `__OPTEE_ALG_ECxx_Py`) hence of meaning. But I agree it's maybe overkillling.
I'm fine removing the condition for the IDs definition.

By this way, I think I should restore all `TEE_ALG_ECDH_P*` being aliases for the unique `TEE_ALG_ECDH_DERIVE_SHARED_SECRET` as it would still make sense.